packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -208,6 +207,20 @@ function DataGrid(props) {
     }
   };
 
+  // The focus is handled by the DataGrid. However, there are times
+  // when the other components may want to change the currently focus
+  // cells. In order to do so, these datagrid methods will be exposed to
+  // allow those components to request focus change.
+  useImperativeHandle(
+    props.focusFuncRef,
+    () => ({
+      setFocusedRowCol,
+      getFocusedCell() { return { row: focusedRow, col: focusedCol }; },
+    }),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [focusedCol, focusedRow],
+  );
```

Comment:
Are we making this more complicated than necessary with the useImperativeHandle hook?